src/store/token-swap/token-swap.thunks.js

```diff
@@ -18,7 +14,8 @@ function fetchAccounts (fromItem) {
     const {
       global: {
         wallet: { publicKeyBase64: hermezAddress },
-        fiatExchangeRatesTask: { data: fiatExchangeRates }
+        fiatExchangeRatesTask: { data: fiatExchangeRates },
```

Comment:
Here you are assuming that the `fiatExchangeRatesTask` is succesful which is not true. Aren't you getting a TypeScript warning here?

Reply:
True, we will need to improve that flow if we continue with token swap but for now `getTokenAmountInPreferredCurrency` has a conditional to check data value

Reply:
But what we can do by the moment is not using object destructuring, and doing just: `fiatExchangeRatesTask,` and check that it's `successful` below.